api/src/main/java/org/openmrs/module/ugandaemrreports/definition/dataset/definition/CustomParametersDatasetDefinition.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+
+package org.openmrs.module.ugandaemrreports.definition.dataset.definition;
+
+import org.openmrs.module.reporting.dataset.definition.BaseDataSetDefinition;
+import org.openmrs.module.reporting.definition.configuration.ConfigurationProperty;
+
+/**
+ * Created by Nicholas Ingosi on 7/19/17.
+ * Custom dataset that will be used to supply additional information in the evaluation contex
+ * such that they can be used on a reporting template just like any other columns
+ */
+public class CustomParametersDatasetDefinition extends BaseDataSetDefinition {
```

Comment:
@ningosi Rename this to GlobalPropertyParametersDatasetDefinition since it only adds Global properties